vespa-feed-client/src/main/java/ai/vespa/feed/client/impl/JettyCluster.java

```diff
@@ -239,11 +239,7 @@ private static String endpointUri(URI uri) {
         return String.format("%s://%s:%s", uri.getScheme(), uri.getHost(), portOf(uri));
     }
 
-    private static class JettyResponse implements HttpResponse {
-        final Response response;
-        final byte[] content;
-
-        JettyResponse(Response response, byte[] content) { this.response = response; this.content = content; }
+    private record JettyResponse(Response response, byte[] content) implements HttpResponse {
```

Comment:
Records are not available on JDK11 - the version used by our internal Hadoop integration.

Reply:
Reverted, but shouldn't this be limited in the pom file so intellij would not suggest it ?

Reply:
Did not fail maven build either....

Reply:
That is unexpected. The build file targets 11 for all clients.